mattermost-10.1.advisories.yaml

```diff
@@ -921,6 +930,10 @@ advisories:
             componentType: go-module
             componentLocation: /usr/bin/mattermost
             scanner: grype
+      - timestamp: 2024-11-18T04:32:37Z
+        type: pending-upstream-fix
+        data:
+          note: The affected component mholt/archiver/v3 does not yet have a fix version, upstream maintainers must implement this.
```

Comment:
Same for this one
```yaml
  - uses: go/bump
    with:
      deps: github.com/mholt/archiver/v3@v3.5.1
      replaces: github.com/mholt/archiver/v3=github.com/anchore/archiver/v3@v3.5.2
```
this could work.